include/aikido/control/SimBarrettFingerPositionCommandExecutor.hpp

```diff
@@ -12,63 +12,68 @@
 namespace aikido {
 namespace control {
 
-/// This executor mimics the behavior of BarretFinger. 
-/// It moves a finger to a desired point; it may stop early if 
+/// This executor mimics the behavior of BarretFinger.
+/// It moves a finger to a desired point; it may stop early if
 /// joint limit is reached or collision is detected.
 /// Only the proximal joint is actuated; the distal joint moves with mimic ratio.
 /// When collision is detected on the distal link, the finger stops.
 /// When collision is detected on the proximal link, the distal link moves
 /// until it reaches joint limit or until distal collision is detected.
-class BarrettFingerPositionCommandExecutor
+class SimBarrettFingerPositionCommandExecutor
 {
 public:
   /// Constructor.
   /// \param _finger Finger to be controlled by this Executor.
   /// \param _proximal Index of proximal dof
   /// \param _distal Index of distal dof
   /// \param _collisionDetector CollisionDetector for detecting self collision
-  ///        and collision with objects. 
+  ///        and collision with objects.
+  /// \param _collideWith CollisionGroup to check collision with fingers.
   /// \param _collisionOptions Default is (enableContact=false, binaryCheck=true,
-  ///        maxNumContacts = 1.) 
-  ///        See dart/collison/Option.h for more information 
-  BarrettFingerPositionCommandExecutor(
+  ///        maxNumContacts = 1.)
+  ///        See dart/collison/Option.h for more information
+  SimBarrettFingerPositionCommandExecutor(
     ::dart::dynamics::ChainPtr _finger, int _proximal, int _distal,
     ::dart::collision::CollisionDetectorPtr _collisionDetector,
+    ::dart::collision::CollisionGroupPtr _collideWith,
     ::dart::collision::CollisionOption _collisionOptions
       = ::dart::collision::CollisionOption(false, 1));
```

Comment:
I suggest making the `_collideWith` argument optional. If it's not specified, default to an empty `CollisionGroup`.